dask_geopandas/io/parquet.py

```diff
@@ -0,0 +1,126 @@
+from functools import partial
+from typing import TYPE_CHECKING
+
+import pandas as pd
+
+import geopandas
+
+import dask.dataframe as dd
+
+try:
+    # pyarrow is imported here, but is an optional dependency
+    from dask.dataframe.io.parquet.arrow import ArrowEngine
+except ImportError:
+    ArrowEngine = object
+
+if TYPE_CHECKING:
+    import pyarrow
+
+
+class GeoArrowEngine(ArrowEngine):
+    @classmethod
+    def read_metadata(cls, *args, **kwargs):
+        meta, stats, parts, index = super().read_metadata(*args, **kwargs)
+
+        # Update meta to be a GeoDataFrame
+        # TODO convert columns based on GEO metadata (this will now only work
+        # for a default "geometry" column)
+        meta = geopandas.GeoDataFrame(meta)
+
+        return (meta, stats, parts, index)
+
+    @classmethod
+    def _arrow_table_to_pandas(
+        cls, arrow_table: "pyarrow.Table", categories, **kwargs
+    ) -> pd.DataFrame:
+        from geopandas.io.arrow import _arrow_to_geopandas
+
+        _kwargs = kwargs.get("arrow_to_pandas", {})
+        _kwargs.update({"use_threads": False, "ignore_metadata": False})
+
+        # TODO support additional keywords
+        return _arrow_to_geopandas(arrow_table)
+
+    @classmethod
+    def _pandas_to_arrow_table(
+        cls, df: pd.DataFrame, preserve_index=False, schema=None
+    ) -> "pyarrow.Table":
+        from geopandas.io.arrow import _geopandas_to_arrow
+
+        # TODO add support for schema
```

Comment:
Should we raise an error if schema is provided in the meantime?

Reply:
Yes, that's a good idea.

(note that those todo's first need a change in geopandas to support those in `_geopandas_to_arrow` and `_arrow_to_geopandas`)